apps/web/src/services/iam/store/user-group-page-store.ts

```diff
@@ -0,0 +1,96 @@
+import { isEmpty } from 'lodash';
+import { defineStore } from 'pinia';
+
+import type { ConsoleFilter } from '@cloudforet/core-lib/query/type';
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+
+import type { ListResponse } from '@/schema/_common/api-verbs/list';
+import type { UserGroupModel } from '@/schema/identity/user-group/model';
+import type { UserListParameters } from '@/schema/identity/user/api-verbs/list';
+import type { UserModel } from '@/schema/identity/user/model';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import type { UserListItemType } from '@/services/iam/types/user-type';
+
+export const useUserGroupPageStore = defineStore('page-user-group', {
+    state: () => ({
+        users: [] as UserListItemType[],
+        selectedUser: {} as UserListItemType,
+        loading: false,
+        groups: [] as UserGroupModel[],
+        searchFilters: [] as ConsoleFilter[],
+        totalCount: 0,
+        selectedIndices: [] as number[],
+        pageStart: 1,
+        pageLimit: 15,
+        modal: {
+            type: '',
+            title: '',
+            themeColor: 'primary',
+            visible: '',
```

Comment:
Question: Why isn't type of `visible` Boolean?

Reply:
I'm using multiple modals and if I used a boolean type, I would have to declare "visible" variables as many times as the number of modals, so I used a string type.